Source/NavierStokesBase.H

```diff
@@ -492,7 +494,9 @@ protected:
     //    NavierStokesBase protected embedded boundary functions              //
     ////////////////////////////////////////////////////////////////////////////
 
+#if !defined(AMREX_USE_CUDA)
 private:
+#endif
```

Comment:
Could just toss this I think.  No biggie.

Reply:
You mean, toss the protected and private ?

Reply:
yeah.  Since it is removed when building for GPUs, why bother having it at all?